1.7/administration/upgrading.md

```diff
@@ -9,6 +9,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 **Important:**
 
+- This document is currently in BETA status. Due to a known issue, this install method will cause all apps and tasks to restart within your cluster.
```

Comment:
@jgarcia-mesosphere @pyronicide should we remove this doc entirely until 1.7.2.?